src/models/model_TimeSeriesTableFullTimeSeriesInner.jl

```diff
@@ -2,17 +2,41 @@
 # Do not modify this file directly. Modify the OpenAPI specification instead.
 
 
-@doc raw"""TimeSeriesTableFull_timeSeries_inner
-
-    TimeSeriesTableFullTimeSeriesInner(;
-        name=nothing,
-        description=nothing,
-        sequence=nothing,
-    )
-
-    - name::String
-    - description::String
-    - sequence::Int64
+"""
+```
+TimeSeriesTableFullTimeSeriesInner(;
+    name=nothing,
+    description=nothing,
+    sequence=nothing,
+)
+```
+  
+This function takes as input the name,the description and the sequence of data e and return a new record
```

Comment:
Could you look at my previous comments for "inner" files?

Reply:
This should be resolved. I rewrote it.